lib/rhc/config.rb

```diff
@@ -63,6 +63,7 @@ class Config
       :ssl_client_cert_file     => [nil,               :path_to_file, 'A client certificate file for use with your server'],
       :ssl_client_key_file      => [nil,               :path_to_file, 'The corresponding key for the client certificate'],
       :ssl_ca_file              => [nil,               :path_to_file, 'A file containing CA one or more certificates'],
+      :ssh                      => [nil,               :path_to_file, 'Path to a ssh executable and command options to use when performing ssh commands'],
```

Comment:
This doesn't make it clear that args need to be quoted with the path to ssh - this isn't obvious to everybody.

Maybe it would be clearer if there were a separate param for ssh options?